docs/Evaluation/Evaluation-Intro.md

```diff
@@ -0,0 +1,112 @@
+# Evaluation
+
+How do we know whether the hack week was successful? What does successful even mean, and how do we define 
+metrics for that success? How can we figure out whether the hack week was successful?
```

Comment:
Remove "How can we figure...." sentence which is a repeat of the first sentence in the para